images/bin/start.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
```

Comment:
@pchico83 @jLopezbarb what's the best way to test changes on this file?

Reply:
You need to create a new `bin` image, build okteto and run `okteto up` with a manifest with secrets.

Reply:
tested!